init.sh

```diff
@@ -45,23 +45,37 @@ function start_zmq_adaptor()
         zmq_adaptor_cmd="${zmq_adaptor_cmd} -d"
     fi
 
-    sudo $zmq_adaptor_cmd >> /dev/stderr &
-    sleep 3s
+    $zmq_adaptor_cmd >> /dev/stderr &
+    sleep 10s
```

Comment:
do 7s make any difference here ? It accounts for large configs ?

Reply:
it accounts for large configs